daemon/api_quotas.go

```diff
@@ -182,6 +182,7 @@ func getQuotaGroupInfo(c *Command, r *http.Request, _ *auth.UserState) Response
 		GroupName:   group.Name,
 		Parent:      group.ParentGroup,
 		Snaps:       group.Snaps,
+		Services:    group.Services,
```

Comment:
A bit magic that nothing on the daemon side needed change except for this line - and the creation still works :)